pythonosc/dispatcher.py

```diff
@@ -54,6 +54,19 @@ def map(self, address, handler, *args, needs_reply_address=False):
     # regarding multiple mappings
     self._map[address].append(Handler(handler, list(args), needs_reply_address))
 
+  def unmap(self, address, handler, *args, needs_reply_address=False):
```

Comment:
this signature feels wrong, there should be no need to reconstruct a handler just to remove it from the map, usually callers will have the handler instance already accessible and they can pass it directly to this method.
Otherwise another cleaner method to get it would be to call `handlers_for_address` and pass the returned handler(s) in the response.

basically: `def unmap(self, address, handler)`

Reply:
Maybe it would make sense to have the map method return the handler? That way you can directly plug it back into unmap. Otherwise I'd suggest having an overloaded method to allow both signatures.
With handlers_for_address you'd have to comb through the handlers again. Imagine having mutliple handlers for the same address but just with different *args arguments. It would add a lot of boilerplate code. Although this admittedly should be a really rare case.